### DISCUSSION TITLE
How to set initalValues in nested dynamic form?

### DISCUSSION THREAD

**Discussion by Lucjus** [ant-design/ant-design#35100] [Q&A] [answered]
Hey, 
How can I set initialValues for nested `Form.List`? 
I want both room and bed added at the start of the app.

https://codesandbox.io/s/trusting-wind-elsdz?file=/src/App.js

**Comment by renbesson** [comment]
Hi Everyone!

I've been trying to do the same for days but no success.

I created a clean sandbox (below) to see if a good soul would help us! 😁

https://codesandbox.io/s/suspicious-leaf-cijswt?file=/src/App.js

**Reply by huntdream** [reply]
https://codesandbox.io/s/optimistic-hypatia-oc7olo?file=/src/App.js
Update your form by `form.setFieldsValue` when your data is ready
cc @Lucjus

**Reply by renbesson** [reply]
That's one option too. I've tried and didn't work for me, tho.
I was able to do it by using the initialValues as below:

```
initialValues={{["ingredients"]: ingredients}}
```

My sandbox from above shows better how to do it.

Thank you for taking the time @huntdream!

**Reply by huntdream** [reply]
Can you provide more detail with your situation? I am curious why it doesn't work

**Reply by renbesson** [reply]
Hi @huntdream.
I just followed the instructions below:
https://ant.design/components/form/#Form.useFormInstance

Probably it didn't work cause my sandbox was using version 4.16 and useFormInstance is available on 4.20.

**Reply by Lucjus** [reply]
I was also able to do it with `initialValues` same as @renbesson 
Thanks a lot!

**Comment by Lucjus** [comment, marked as answer]
ex: `initialValues={{["ingredients"]: ingredients}}` is a way to do it.
